app/jobs/bulkrax/create_relationships_job.rb

```diff
@@ -37,11 +37,11 @@ def perform(parent_identifier:, importer_run_id:)
       end.sort_by(&:order)
 
       @importer_run_id = importer_run_id
-      @parent_record = find_record(parent_identifier)
+      @parent_entry, @parent_record = find_record(parent_identifier, importer_run_id)
```

Comment:
example use case that made this necessary:
a collection was imported in importer 1, and in importer 2 we are importing a work to add to that collection. in that case, line 56 below would have failed because this importer_run doesn't have the collection.